tasmota/berry/include/be_gpio_defines.h

```diff
@@ -358,7 +358,10 @@ const be_const_member_t lv_gpio_constants[] = {
     { "ZEROCROSS", (int32_t) GPIO_ZEROCROSS },
     { "ZIGBEE_RST", (int32_t) GPIO_ZIGBEE_RST },
     { "ZIGBEE_RX", (int32_t) GPIO_ZIGBEE_RX },
-    { "ZIGBEE_TX", (int32_t) GPIO_ZIGBEE_TX },
+    { "ZIGBEE_TX", (int32_t) GPIO_ZIGBEE_TX },    
```

Comment:
Read the header of this file:

* The keys in the array belox must be in lexicographic order

So add your changes at the correct place.